code/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
+import React from 'react';
+import logo from './assets/logo2.png'
+
+const Header = () => {
+  return (
+    <header>
+      <div className="image-wrapper">
+        <img className="logo" src={logo} alt="Happy Thoughts" />
```

Comment:
The Happy Thoughts logo is very cute. It makes it nice and clean and reusable to have a header component.